backend/backoffice/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from backoffice.models import TalkEntry
+
+
+@admin.register(TalkEntry)
+class TalkEntryAdmin(admin.ModelAdmin):
+    search_fields = ("name", "title")
+    ordering = ("name", "date", "title")
+    list_display = ("name", "date", "title")

```

Comment:
I suggest to display the date first, as it is used as the main ordering.

We can also add more columns like the `display` state.

```suggestion
    list_display = ("date", "name", "title", "youtube_link",  "display")
```

Before merging we will transform the YouTube link in a real HTML link.

Reply:
It's adding this button that redirects directly to the link. Notes for later at the bottom of Pull Request